packages/driver/cypress/e2e/commands/cookies.cy.js

```diff
@@ -156,7 +156,7 @@ describe('src/cy/commands/cookies', () => {
           expect(lastLog.get('state')).to.eq('failed')
           expect(lastLog.get('name')).to.eq('getCookies')
           expect(lastLog.get('message')).to.eq('')
-          expect(err.message).to.eq('`cy.getCookies()` timed out waiting `50ms` to complete.')
+          expect(err.message).to.eq('Timed out retrying after 50ms: `cy.getCookies()` timed out waiting `50ms` to complete.')
```

Comment:
Message is kind of redundant? 🤷‍♂️  The added message is from applying retry logic to cookies.

Reply:
I wonder if we want to do something like this:

```suggestion
          expect(err.message).to.eq('Timed out retrying after 50ms: `cy.getCookies()` failed to complete.')
```

Reply:
Presents better imo. Makes sense, and if users run into this error we should almost immediately know where it is.